infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/deployment.yaml

```diff
@@ -29,8 +29,6 @@ spec:
         image: "{{ .Values.global.containerRegistry}}/{{ .Values.image }}:{{ .Values.global.appImageTag }}"
         imagePullPolicy: Always
         envFrom:
-          - configMapRef:
```

Comment:
There is no more `api-config`. All api pods now rely solely on the security config map